### PR TITLE
Added Python-Code for preforming ZIP

### PR DESCRIPTION
**Pull request by gabru-md**
Python Code is added for generating ZIPs and with few minor tweaks of other files.
All the OS commands are replaced with python library : zipfile
Tested and Works!
Will add screenshots of the app!

It is a subsequent fix for https://github.com/fossasia/badgeyay/issues/76

Fixes #

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [X] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Replaced OS commands
- Added Python Code
- tweaked the index.html and main.py for the same.

### CODE REVIEW

**Comment by gabru-md** [comment]
Screen-Shots for the application: 

b3

b2

b1

PS : @jajodiaraghav @S2606 @niccokunzmann

**Comment by jajodiaraghav** [comment]
@gabru-md We do not want to remove the linux commands in merge_badges.py
Can you please remove all the changes from merge_badges.py file and add the code to merge_badges_v2.py?

File: app/merge_badges.py [hunk 1]
```diff
@@ -25,6 +26,18 @@
 input_folders = [file for file in os.listdir(BADGES_FOLDER)
                if file.lower().endswith(".badges")]
 
+def _zipFile(src,dst,allowed):
+    print(dst)
```
Comment (jajodiaraghav) [inline app/merge_badges.py]: Please remove this line

File: app/merge_badges.py [hunk 2]
```diff
@@ -60,7 +73,7 @@ def generate_pdfs(folder_path):
 	print ("Created {}".format(final_path))
 	print ("Generating ZIP file")
 
-	os.system('find static/badges/ \( -iname \*.svg -o -iname \*.pdf \) | zip  -@ static/badges/all-badges.zip')
+	_zipFile("static/badges","static/badges/all-badges",["svg","pdf"])
```
Comment (jajodiaraghav) [inline app/merge_badges.py]: Put a space after comma

File: app/merge_badges.py [hunk 3]
```diff
@@ -69,5 +82,5 @@ def generate_pdfs(folder_path):
 
 	for folder in input_folders:
 		folder_path = os.path.join(BADGES_FOLDER, folder)
-		file_name = folder_path + '.svg.zip'
-		os.system('find ' + folder_path + ' -type f -name *.svg | zip -@ ' + file_name)
+		file_name = folder_path.replace('.','-') + '-svg'
+		_zipFile(folder_path,os.path.join(folder_path,file_name),["svg"])
```
Comment (jajodiaraghav) [inline app/merge_badges.py]: Put a space after comma

**Comment by jajodiaraghav** [comment]
Looks Good!
Made a few comments :)
We can merge this 🚀

File: app/merge_badges.py (hunk 1 above)
Comment (gabru-md) [reply app/merge_badges.py]: sure!

File: app/merge_badges.py (hunk 2 above)
Comment (gabru-md) [reply app/merge_badges.py]: okay

File: app/merge_badges.py (hunk 3 above)
Comment (gabru-md) [reply app/merge_badges.py]: okay

Comment (gabru-md) [reply app/merge_badges.py]: adding a space after every comma

**Comment by gabru-md** [comment]
@jajodiaraghav Please review

**Comment by jajodiaraghav** [comment]
Thanks @gabru-md

**Pull request merged**